### PR TITLE
add project Music-Wiki

### PR DESCRIPTION
**Pull request by ZyanWlayor**
add project Music-Wiki

### CODE REVIEW

**Comment by cwaring** [comment]
Hi @ZyanWlayor. I could not find an registration for this entry, have you submitted this project at: https://slingshot.filecoin.io/register-now/ ?

**Comment by cwaring** [comment]
Sorry ignore the above. I now see your entry in the queue waiting import, this will be processed today. Thanks

File: participants/Music-Wiki.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+## (3) What does your application/UI do?
+
+*The website can receive music uploaded by people.Everyone can improve our music catalogue,everyone can find search music through name.*
+
+## (4) If your project is using a curated dataset, which one are you using?
+
+*redis and mysql*
+
+## (5) If your project is not using a curated dataset, please tell us a little bit more about your data by answering the questions below.
+
```
Comment (cwaring) [inline participants/Music-Wiki.md:26]: Do you have the rights to this data or is all the music uploaded copyright free?

Comment (ZyanWlayor) [reply participants/Music-Wiki.md:26]: There is no complete data in our application, and the data uploaded to the IPFS network will be reviewed by another copyritht checking model, later.

**Review by dineshshenoy — APPROVED** [review]

**Pull request merged**